Sources/DolbyIORTSCore/State/StateMachine.swift

```diff
@@ -168,6 +138,10 @@ final actor StateMachine {
 
     func onVideoTrack(_ track: MCVideoTrack, withMid mid: String) {
         switch currentState {
+         case .subscribing:
+             let state = SubscribedState()
+             state.addVideoTrack(track, mid: mid)
+             currentState = .subscribed(state)
```

Comment:
I think it was okay to ignore the first video and audio tracks that we receive as they are fired before the onSubscribed and onActive events. The reason is we do an addRemoteTrack call when we have the source and track information on `onActive` event. This addRemoteTrack call gives us back the video and audio track for the main source anyways.
What were you thoughts in using the tracks that arrive before we have the source information or source id ?

Reply:
yes, addressed